staker/_TEST_staker_rpc_get_test.gnoa

```diff
@@ -35,18 +35,18 @@ func init() {
 	poolTiers["ATOM/WETH_500"] = 3
 }
 
-func TestPoolInitCreatePool(t *testing.T) {
-	std.TestSetPrevAddr(gsa)
-	pl.InitManual()
-	std.TestSkipHeights(1)
-
+func TestPoolCreatePool(t *testing.T) {
 	std.TestSetPrevAddr(test1)
+	for i := 0; i < (5 * 1); i++ {
```

Comment:
Is there any reason that using `5 * 1` instead of `5`? 🤔🤔🤔

Reply:
Basically it uses `5 x {num_of_pool_to_create}` template for every tests. It would be better to keep its format